### PR TITLE
#32 Add a launcher to dump the result of CS mode in a file

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: sources/Launcher/DYNComputeSimulationLauncher.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline sources/Launcher/DYNComputeSimulationLauncher.cpp:2]: Just 2021 ?

File: sources/Launcher/CMakeLists.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 set(DYN_ALGO_LAUNCHER_SOURCES
   DYNComputeLoadVariationLauncher.cpp
+  DYNComputeSimulationLauncher.cpp
```
Comment (gautierbureau) [inline sources/Launcher/CMakeLists.txt:14]: .h is not necessary to install ? in DYN_ALGO_LAUNCHER_HEADERS below

File: sources/Launcher/DYNComputeSimulationLauncher.h [hunk 3]
```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline sources/Launcher/DYNComputeSimulationLauncher.h:2]: same

File: sources/Launcher/DYNComputeSimulationLauncher.cpp (hunk 1 above)
Comment (rosiereflo) [reply sources/Launcher/DYNComputeSimulationLauncher.cpp]: solved

File: sources/Launcher/CMakeLists.txt (hunk 2 above)
Comment (rosiereflo) [reply sources/Launcher/CMakeLists.txt:14]: solved

File: sources/Launcher/DYNComputeSimulationLauncher.h (hunk 3 above)
Comment (rosiereflo) [reply sources/Launcher/DYNComputeSimulationLauncher.h]: solved

**Comment by gautierbureau** [comment]
The branch now contains already merged commits I think. A merge instead of a rebase was done I presume

**Pull request closed without merging**